### PR TITLE
[Templates] Add condition for "form last updated" field

### PR DESCRIPTION
**Pull request by ncksllvn**
## Description

Issue - https://github.com/department-of-veterans-affairs/va.gov-team/issues/14024

## Testing done
Tested against a page where "issue date" is the latest date and against a page where "revision date" is the latest date

### Example where "issue date" is latest date
http://localhost:3001/preview?nodeId=5748
image

### Example where "revision date" is latest date
http://localhost:3001/preview?nodeId=5776
image

## Acceptance criteria
- [ ] "form last updated" shows the latest date between revision date and issue date

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/layouts/va_form.drupal.liquid [hunk 1]
```diff
@@ -50,13 +50,18 @@
...
 
-          {% if fieldVaFormRevisionDate %}
+          {% if fieldVaFormRevisionDate or fieldVaFormIssueDate %}
             <div>
               <dt class="vads-u-font-weight--bold vads-u-display--inline">
                 Form last updated:
               </dt>
               <dd class="vads-u-display--inline">
-                {{ fieldVaFormRevisionDate.value | humanizeDate }}
+                {% assign revisionDateIsLatestUpdate = fieldVaFormRevisionDate.value | isLaterThan: fieldVaFormIssueDate.value %}
```
Comment (ncksllvn) [inline src/site/layouts/va_form.drupal.liquid:59]: you can't use a liquid filter in an if-statement so i had to assign it to a var beforehand

File: src/site/filters/liquid.unit.spec.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+describe('isLaterThan', () => {
+  it('returns true when the left arg is a timestamp later than the right arg', () => {
+    expect(liquid.filters.isLaterThan('2020-01-11', '2016-07-10')).to.be.true;
+  });
+
+  it('returns false when the left arg is a timestamp before the right arg', () => {
+    expect(liquid.filters.isLaterThan('2016-12-11', '2017-01-12')).to.be.false;
+  });
+});
```
Comment (ncksllvn) [inline src/site/filters/liquid.unit.spec.js:16]: unit tests for liquid filters is overdue

**Review by rianfowler — APPROVED** [review]

File: src/site/filters/liquid.unit.spec.js (hunk 2 above)
Comment (kelsonic) [reply src/site/filters/liquid.unit.spec.js:16]: so true! 🔥

**Review by kelsonic — APPROVED** [review]
Looks great! 🎉

**Pull request merged**